tests/conftest.py

```diff
@@ -14,22 +14,43 @@
 
 from httpcore._types import URL
 
+from .marks.curio import curio_pytest_pycollect_makeitem, curio_pytest_pyfunc_call
+
 PROXY_HOST = "127.0.0.1"
 PROXY_PORT = 8080
 
 
+def pytest_configure(config):
+    # register an additional marker
```

Comment:
(Nit) I think the code is self-explanatory here, we don't really need this comment (I think it came from the pytest docs example anyway?).

```suggestion
```